test/Debugger/MultipleContextStack.js

```diff
@@ -21,8 +21,8 @@ var script1 = WScript.LoadScript("\
   function Script1Func3() { scriptFunc3(); } \
   function setFunc2(func) { scriptFunc2 = func; } \
   function setFunc3(func) { scriptFunc3 = func; }",
-  "samethread", "Script1.js");

```

Comment:
When file name is given, LoadScript is trying to find the full path on xplat and that fails